src/Components/DeleteModal/DeleteModal.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import '../../styles/Modal.css';
+import '../../styles/Button.css';
+import './DeleteModal.css';
+
+export default function DeleteModal({
+  token,
+  itemName,
+  setDeleteItemName,
+  deleteItem,
+}) {
+  const hideModal = () => {
+    setDeleteItemName('');
+  };
+
+  const confirmDelete = () => {
+    deleteItem(token, itemName);
+    hideModal();
+    alert(`"${itemName}" has been deleted from your list.`);
```

Comment:
Not a blocker, just a 'if you want' suggestion. Since we already have a modal showing, could we maybe update the wording instead of adding an alert on top of it?

Reply:
Good idea! We decided to create a separate issue to make an accessible, reusable Modal component as the current deleteModal component is hardcoded to the delete button, and we fear that refactoring right now will break our code for this issue.